lib/org/chromium/webidl/CanonicalCollection.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  package: 'org.chromium.webidl',
+  name: 'CanonicalCollection',
+
```

Comment:
Please document the purpose of this class and the context it's intended for. In particular, we shove definitions into the same data class (I think) because of the transactional nature of canonicalization. Is that right?

It's also worth mentioning why this data-only class has no id: It's supposed to have its definitions unpacked into storable definitions, right?